### PR TITLE
fix(report): align printResult to output the correct total requests

### PR DESCRIPTION
**Pull request by elirandav**
Closes https://github.com/mcollina/autocannon/issues/228

### CODE REVIEW

**Comment by mcollina** [comment]
I'm not sure this is the correct fix, as the additional requests are done. This PR is just hiding them.

**Comment by elirandav** [comment]
> I'm not sure this is the correct fix, as the additional requests are done. This PR is just hiding them.

correct @mcollina . 
we need to use the `result` and not `opts`. I aligned the PR. 

In the scenario I am fixing, all the requests were sent to the tested server, only the output was wrong: 10 requests instead of 20 requests. I verified it with my local server - it receives all 20 requests.

**Review by mcollina — APPROVED** [review]
Lgtm

File: test/printResult.test.js [hunk 1]
```diff
@@ -48,3 +49,251 @@ test('should stdout (print) the result', (t) => {
...
+      p90: 3319,
+      p97_5: 3319,
+      p99: 3319,
+      p99_9: 3319,
+      p99_99: 3319,
+      p99_999: 3319
+    }
+  }
+  const opts = {
+    outputStream: {
```
[the remark this answers is not in the document]
Comment (elirandav) [reply test/printResult.test.js:154]: in the test, I am simulating the bug in method `printResult`, which gets two parameters as input: `result` and `opts`. 

so to gets the same behaviour while executing `node autocannon...`, I took the two objects value while debugging and used them as input for simulating the bug in the test.

[the remark this answers is not in the document]
Comment (mcollina) [reply test/printResult.test.js:154]: I don't think this entire `opts` object is needed, could you remove the properties that are not used?

[the remark this answers is not in the document]
Comment (elirandav) [reply test/printResult.test.js]: right. I removed the non-mandatory input. it seems `throughput` is mandatory with all its pX properties.

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**